README.md

```diff
@@ -71,11 +98,23 @@ You can access the client through `this.$jsonApi` or `context.$jsonapi`.
 
 **Example:**
 
+- AsyncData hook :
+
+```js
+async asyncData({ $jsonApi }) {
+    articles = await $jsonApi.get('/article')
+
+    return {
+        articles,
+    }
+  }
+```
+
+- Fetch hook
+
 ```js
 async fetch() {
-    this.articles = await this.$jsonApi.get('/article').then(articles => {
-      return articles.data
-    })
+    this.articles = await this.$jsonApi.get('/article')
```

Comment:
The changes are not equivalent since the current example returns the `data` property of `articles`.